Partial fix #2765: Another tranche of stale deprecation removals

LeeTibbert opened the pull request:
This PR is a partial and not total fix of Issue #2765. Could that issue please remain open?
When the deprecation removal PRs of his series either merge or are declined, I need to
take one or more runs through the entire code base looking for more deprecated
items whose time for removal has come.

This PR remove 10 more deprecated items.

The deprecated items are documented in `changelog/0.5.0.md`  in an unusual, temporary 
way. PR #3003 makes some changes to the same file in the same area.  My intent is to
avoid git rebase  hassles.  I will merge the two sections after the dust settles.

WojciechMazur commented on docs/changelog/0.5.0.md:
```diff
@@ -152,9 +151,9 @@ default to the project name.
-    nativeConfig ~= { c =>
-      c.withBasename("myapp")
-    }
+	nativeConfig ~= { c =>
+	  c.withBasename("myapp")
+	}
```
Looks like spaces were replaced with tabs in this file.

LeeTibbert commented:
Wojciech,

Thank you for the review.  

I fixed the unintended white space change. 

Now that PR #3003 has merged, I re-based the `changelog/0.5.0.md` changes of this
PR onto mainline & re-edited. 

I also changed the release year for 0.5.0 to 2023, unless you have a holiday surprise 
in store. One thing fewer to be done later.

LeeTibbert commented:
Windows runtime test timed out after 6 hours. All others are Green.
